src/main/java/launutsapi/LaunutsController.java

```diff
@@ -1,8 +1,155 @@
 package launutsapi;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.simple.JSONArray;
+import org.json.simple.parser.ParseException;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @RestController
 public class LaunutsController {
+	
+	@Autowired
+	private NutsService nut_service;
+	
+	@Autowired
+	private LauService lau_service;
+	
+	private Nuts nut;
+	private Lau lau;
+	
+	private String notification = "Please enter query in correct format."
+		     + "Some correct formats are like these "
+		     + "http://localhost:8080/launuts/nuts/DEA47/json, "
+		     + "http://localhost:8080/launuts/nuts/DEA47/ttl, "
+		     + "http://localhost:8080/launuts/nuts/paderborn/json, "
+		     + "http://localhost:8080/launuts/nuts/paderborn/ttl, "
+		     + "http://localhost:8080/launuts/nuts/ttl, "
+		     + "http://localhost:8080/launuts/nuts/json ";
+	
+	@RequestMapping("/launuts/nuts/{query_string}/json")
+	public List<Nuts> getNutsJson(@PathVariable String query_string) throws IOException, ParseException {
+		
+		//Validate input: If user trying to enter a nut id
+		if(query_string.matches(".*\\d.*") && !(query_string.toLowerCase().matches("^((de)|(de\\D))((\\d{1})|(\\d{2})|(\\d{3}))$"))) {
+	
+			nut = new Nuts(notification);	
+			return Arrays.asList(nut);	
+		}
+		else 
+			return Arrays.asList(nut_service.getNutsJson(query_string));	
+			
+	}
+	
+	@RequestMapping("/launuts/lau/{query_string}/json")
```

Comment:
In this API, for every lau_code, it is generating same LAU. I have also attached image below for reference.
![error2](https://user-images.githubusercontent.com/52550720/84090650-ad20a900-a9f2-11ea-9123-920b63395b2f.JPG)

Reply:
Thanks for catching this. It is resolved now.

Reply:
Ok, It is working only for english words but not for German words. For example: words with umlaut. Please see the below picture for reference.
![no_umlauts](https://user-images.githubusercontent.com/52550720/84567175-329bb480-ad77-11ea-978b-e93dc6aff77c.JPG)

Reply:
Working fine for me!!

![image](https://user-images.githubusercontent.com/39089286/84573997-237f2b80-ada4-11ea-8734-91f30d075797.png)